### PR TITLE
Create S4654830-GCN Model

### PR DESCRIPTION
**Pull request by Alanlxc**
This project's purpose is to create a suitable multi-layer GCN model to carry out a semi-supervised multi-class node classification using Facebook Large Page-Page Network data set [2] with reasonable accuracy. (You may use this partially processed dataset where the features are in the form of 128 dim vectors.) You should also include a TSNE embeddings plot with ground truth in colors.

Following file structure :
Readme.md
Data document
Data preprare.py
GCN build.py
Training.py
Predict.py
Visualization.py

### CODE REVIEW

**Comment by Alanlxc** [comment]
Data resource link
[https://snap.stanford.edu/data/facebook-large-page-page-network.html](url)

**Comment by Alanlxc** [comment]
implement data folder into code

**Comment by Alanlxc** [comment]
Understand graph data and find the class version

**Comment by Alanlxc** [comment]
Fix bug in data type problem, and normalize the data to 1

**Comment by Alanlxc** [comment]
Data hyperparameter setting completed

**Comment by Alanlxc** [comment]
Training data and validation data and test data split complete
